Gems/Atom/Tools/MaterialEditor/Code/Source/Viewport/MaterialViewportComponent.cpp

```diff
@@ -429,4 +429,52 @@ namespace MaterialEditor
             ReloadContent();
         });
     }
+
+    void MaterialViewportComponent::OnCatalogAssetChanged(const AZ::Data::AssetId& assetId)
+    {
+        auto ReloadLightningAndModelPresets = [this, &assetId](AZ::Data::AssetCatalogRequests* assetCatalogRequests)
```

Comment:
typo, this should be lighting not lightning

Reply:
Will fix